src/backend/distributed/utils/node_metadata.c

```diff
@@ -264,52 +273,160 @@ master_remove_node(PG_FUNCTION_ARGS)
 Datum
 master_disable_node(PG_FUNCTION_ARGS)
 {
-	const bool onlyConsiderActivePlacements = true;
 	text *nodeNameText = PG_GETARG_TEXT_P(0);
 	int32 nodePort = PG_GETARG_INT32(1);
-
 	char *nodeName = text_to_cstring(nodeNameText);
+	WorkerNode *workerNode = ModifiableWorkerNode(nodeName, nodePort);
 	bool isActive = false;
+	bool onlyConsiderActivePlacements = false;
 
-	WorkerNode *workerNode = NULL;
+	if (WorkerNodeIsPrimary(workerNode))
+	{
+		/* Delete refrence table placements so they are not taken into account for
```

Comment:
refrence -> reference.

Our usual commenting format is
```c
/*
 * Delete ..
 * the ..
 */
```

Reply:
not resolved

Reply:
Seems that comment was in two places and I only changed it in one of them.